dashboard/app/controllers/courses_controller.rb

```diff
@@ -7,6 +7,7 @@ def index
     @recent_courses = current_user.try(:recent_courses)
     @is_teacher = !!(current_user && current_user.teacher?)
     @is_english = request.language == 'en'
+    @is_signed_out = current_user.nil?
```

Comment:
Nit: I wonder if this will be easier to reason about if we pass this around as `is_signed_in` (and then display our extra UI `if !isSignedIn`